docs/community/open-source-community/contributors.md

```diff
@@ -24,15 +24,15 @@ There is one other, important, way to contribute: **use the work in the Baseline
 
 The Baseline Protocol Code of Conduct is located here
 
-{% embed url="https://github.com/ethereum-oasis/baseline/blob/master/CODE_OF_CONDUCT.md" %}
+{% embed url="https://github.com/ethereum-oasis-op/baseline/blob/master/CODE_OF_CONDUCT.md" %}
```

Comment:
main branch is no longer called master. (But also, maybe change the previous line...)

```suggestion
{% embed url="https://github.com/ethereum-oasis-op/baseline/blob/main/CODE_OF_CONDUCT.md" %}
```